nemo_skills/code_execution/sandbox.py

```diff
@@ -116,17 +116,17 @@ def __init__(
         )
         self.ssh_server = os.getenv("NEMO_SKILLS_SSH_SERVER", ssh_server)
         self.ssh_key_path = os.getenv("NEMO_SKILLS_SSH_KEY_PATH", ssh_key_path)
-        # will keep state of code sessions
-        self.sessions = {}
-
-    def clear_session(self, session_id):
```

Comment:
instead of removing this, should we call it inside code execution model when we are done with the current session? And change this function to actually delete session from the sandbox (there is some new api for that). This way we can avoid clogging the cpus with inactive sessions? Or does it happen like this in some other way already?